avaframe/out3Plot/outQuickPlot.py

```diff
@@ -121,7 +121,7 @@ def generatePlot(dataDict, avaName, outDir, cfg, plotDict):
     ax[1].legend()
     fig.savefig(os.path.join(outDir, 'Profiles_%s_%s.%s' % (avaName, simName, outputFormat)))
 
-    log.info('Figures saved to: %s' % outDir)
+    log.debug('Figures saved to: %s' % outDir)
```

Comment:
mh, maybe leave this on level info? This is something the user might look for quite often...

Reply:
Changed to info level